test/src/mbgl/test/fixture_log_observer.cpp

```diff
@@ -60,6 +60,7 @@ size_t FixtureLog::Observer::count(const Message& message, bool substring) const
 
 FixtureLog::FixtureLog() : observer(new FixtureLogObserver(this)) {
     Log::setObserver(std::unique_ptr<Log::Observer>(observer));
+    Log::useLogThread(false);
```

Comment:
Is there a test where useThread is true?

Reply:
All other tests that don't count the number of log messages have `useThread == true`